Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm

```diff
@@ -1068,6 +1071,11 @@ - (BOOL)isAccessibilityElement
     return m_isAccessibilityElement;
 }
 
+- (void)_clearCachedIsAccessibilityElementState
+{
+    m_isAccessibilityElement = -1;
```

Comment:
I know this was pre-existing, but could we just make this a tristate enum or at least have some constants? I hate magic numbers and sooner or later one of us will have to debug it and hunt around figuring out what -1 means